src/app/components/SubjectHeading/SubjectHeading.jsx

```diff
@@ -148,14 +178,18 @@ class SubjectHeading extends React.Component {
     } = this.addEmphasis(label);
 
     return (
-      <li data={`${subjectHeading.uuid}, ${container}`}>
-        <a className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""}` + `${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
+      <li data={`${subjectHeading.uuid}, ${container}`} className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""}`}>
+        <a  className={`subjectHeadingRow ${ open || children ? "openSubjectHeading" : ""} + ${this.props.nested ? ' nestedSubjectHeading' : ''}`} >
           <span style={{'paddingLeft': `${20*indentation}px`}} className="subjectHeadingLabelAndToggle">
             <span onClick={container !== 'context' ? this.toggleOpen : () => {} } className="subjectHeadingToggle" >{desc_count > 0 ? (!open ? '+' : '-') : null}</span>
             <span className="subjectHeadingLabel" onClick={this.linkToShow}><span>{rest}</span>{rest === '' ? '' : ' -- ' }<span className='emph'>{emph}</span></span>
           </span>
           <span className="subjectHeadingAttribute titles">{`${bib_count}`}</span>
           <span className="subjectHeadingAttribute narrower">{`${desc_count}`}</span>
+          { open && narrower.length > 1 && uuid.length > 0 && (container !== 'context' || location.pathname.includes(uuid))
```

Comment:
Are these conditions not redundant at all? Seems like there could be a clearer source of truth

Reply:
Well this check could be refactored into a separate method, but as far as I know these are all independent of each other. I could add some comments as to what each of these checks is doing if that would be helpful.